autoPyTorch/pipeline/components/setup/preprocessor/utils.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, List
+
+import numpy as np
+
+import torchvision.transforms
+
+from autoPyTorch.pipeline.components.preprocessing.base_preprocessing import autoPyTorchPreprocessingComponent
+
+
+def get_preprocess_transforms(X: Dict[str, Any]) -> torchvision.transforms.Compose:
+    transforms = list()  # type: List[autoPyTorchPreprocessingComponent]
+    for key, value in X.items():
+        if isinstance(value, autoPyTorchPreprocessingComponent):
+            transforms.append(value)
+
+    return torchvision.transforms.Compose(transforms)
+
+
+def preprocess(dataset: np.ndarray, transforms: torchvision.transforms.Compose) -> np.ndarray:
+    # uncomment next 2 lines if dataset is not np array
+    # dataset = copy.copy(dataset)
+    # dataset.data = transforms(dataset.data)
```

Comment:
This looks like tensor, so it makes me wonder... instead of having this hear, does it make sense to create an issue to support gpu preprocessing later?